lib/create-contentful-api.js

```diff
@@ -153,6 +157,71 @@ export default function createClientApi ({ http }) {
       .then((response) => wrapOrganizationCollection(http, response.data), errorHandler)
   }
 
+  /**
+   * Gets a collection of usage periods for the organization. Only available to Admin or owner of the organization.
+   * @memberof ContentfulClientAPI
+   * @param organizationId
+   * @return {Promise<UsagePeriodCollection>} Promise for a collection of usage periods
```

Comment:
`UsagePeriodCollection` needs to be define in the jsdoc comment on top similar to sys etc..

```
/**
 * @memberof UsagePeriodCollection
 * @typedef UsagePeriodCollection
 * @prop {number} total
 * @prop {number} skip
 * @prop {number} limit
 * @prop {Array<UsagePeriod.UsagePeriodCollection>} items
 * @prop {function(): Object} toPlainObject() - Returns this UsagePeriod collection as a plain JS object
 */
```

Reply:
@Khaledgarbaya the definitions for these are in the entity files (`entities/usage.js` and `entities/usagePeriod.js`). Would you prefer they go into this file instead?

Also, I was previously missing the definitions of the namespace, but once I added those, links to types (Usage, UsageCollection, UsagePeriod, UsagePeriodCollection) is working now.